plugin.js

```diff
@@ -0,0 +1,50 @@
+const AdminBro = require('admin-bro')
+
+module.exports = {
+  name: 'AdminBroExpressjs',
+  version: '0.1.0',
+
+  register: async (app, options) => {
+    const express = require('express')
+    const router = express.Router()
+    const path = require('path');
+    const bodyParser = require('body-parser');
+
+    const { routes, assets } = AdminBro.Router
+    const admin = new AdminBro(options)
+
+    app.use(bodyParser.json())
+    app.use(bodyParser.urlencoded({ extended: true }))
+
+    routes.forEach((route) => {
+      const expressPath = route.path.replace(/{/g, ':').replace(/}/g, '')
```

Comment:
I would add simple code comment above like this:
// we have to change routes defined in admin bro from xxx to yyyy